tests/testapp/tests/test_selenium.py

```diff
@@ -143,3 +147,30 @@ def test_shelf_add_form_organization_field(self):
             self.assertEqual(len(org_select.all_selected_options), 1)
             self.assertEqual(org_select.first_selected_option.text, org1.name)
         self.logout()
+
+    def test_organization_autocomplete_filter(self):
+        """
+        The autocomplete_filter should show option to filter
+        shared objects to non-superuser.
```

Comment:
I am not sure I fully understand this docstring/comment: should the option to filter shared objects appear only to users who have the is_superuser flag set to False? Or what does this text mean exactly? Let's make it unambiguous.